process/sync/shardForkDetector.go

```diff
@@ -16,6 +16,7 @@ var _ process.ForkDetector = (*shardForkDetector)(nil)
 // shardForkDetector implements the shard fork detector mechanism
 type shardForkDetector struct {
 	*baseForkDetector
+	doJobOnBHProcessedFunc func(_ data.HeaderHandler, _ []byte, _ []data.HeaderHandler, _ [][]byte)
```

Comment:
also here, could have kept the parameter names

Reply:
Done